test/validation/api/system/system.c

```diff
@@ -517,9 +517,37 @@ static void system_test_info(void)
 			printf("        ODP_CPU_ARCH_ARMV8_5\n");
 		else if (info.cpu_isa_sw.arm == ODP_CPU_ARCH_ARMV8_6)
 			printf("        ODP_CPU_ARCH_ARMV8_6\n");
+		else if (info.cpu_isa_sw.arm == ODP_CPU_ARCH_ARMV8_6)
+			printf("        ODP_CPU_ARCH_ARMV8_6\n");
```

Comment:
`ODP_CPU_ARCH_ARMV8_6 ` is now checked twice. Otherwise looking good, you can add my reviewed-by tag while you fix this.

Reply:
Sorry for the typo. Corrected it now. Thanks.